clang-tools-extra/include-cleaner/test/tool.cpp

```diff
@@ -48,3 +48,13 @@ int x = foo();
 //        RUN: clang-include-cleaner -edit --ignore-headers="foobar\.h,foo\.h" %t.cpp -- -I%S/Inputs/ 
 //        RUN: FileCheck --match-full-lines --check-prefix=EDIT2 %s < %t.cpp
 //  EDIT2-NOT: {{^}}#include "foo.h"{{$}}
+
+//        RUN: mkdir -p $(dirname %t)/out
```

Comment:
you can use `%T` instead of `dirname %t`, you can find full list in https://llvm.org/docs/CommandGuide/lit.html#substitutions, same below.

also it's safer to run a `rm -rf %T` beforehand to make sure we're starting clean

Reply:
It turns out `%T` is also deprecated. I replaced my shell invocation with `%t.dir`, as this is the recommended method, as described in [https://reviews.llvm.org/D69572](https://reviews.llvm.org/D69572). I also slightly modified the test invocation so that it works on Windows as well. (Tested on a local Windows machine.)